components/pedigree/resources/src/main/resources/pedigree/view/abstractHoverbox.js

```diff
@@ -267,6 +270,25 @@ define([
                 this.generateHandles();
         },
 
+        setDoNotHideHoverbox: function(raphaelElement) {
+            // raphaelElement is either a set which has .forEach() defined (but has no .node),
+            // or a single element which has .node but not .forEach()
+            if (raphaelElement.node) {
+                raphaelElement.node.setAttribute('class', 'donothidehoverbox ' + raphaelElement.node.getAttribute('class'));
+                // a hack for raphael <tspan> complication when using SVG (on most modern browsers)
+                if (raphaelElement.node.children &&
+                    raphaelElement.node.children.length > 0 &&
+                    raphaelElement.node.children[0].nodeName == "tspan") {
+                    raphaelElement.node.children[0].setAttribute("class","donothidehoverbox");
+                }
+            } else {
+                var _this = this;
+                raphaelElement.forEach(function(element) {
+                    _this.setDoNotHideHoverbox(element);
+                });
+            }
+        },
```

Comment:
Not a huge problem with so many elements with that class name, just wondering if there's a more efficient way, e.g. by grouping elements semantically. But this is definitely fine now (hence the approval of the PR).